closed/Components.gmk

```diff
@@ -0,0 +1,71 @@
+# ===========================================================================
+# (c) Copyright IBM Corp. 2020, 2022 All Rights Reserved
```

Comment:
Please ensure correct copyright dates are correct in all modified files (the second year should be 2023).